vars/bakeAMI.groovy

```diff
@@ -76,6 +76,7 @@ def call(body) {
       '''
       sh '''#!/bin/bash
       AMI_BUILD_ID=${BRANCH}-${AMI_BUILD_NUMBER}
+      export OPT_VARS="-var ami_users=${AMI_USERS}"
```

Comment:
what happens if this this is a null value? Would this be better off being added to the bakery script with the rest of the options?